src/client/components/BaseLayout/Masthead.tsx

```diff
@@ -11,6 +15,14 @@ const useStyles = makeStyles(() =>
       padding: '4px 0',
       fontSize: '14px',
     },
+    mastheadsticky: {
+      zIndex: 2,
+      position: 'absolute',
+      backgroundColor: '#f0f0f0',
+      height: 'auto',
+      padding: '4px 0',
+      fontSize: '14px',
+    },
```

Comment:
my observation here is that the only reason we had to declare this is to have a different `position` value. In this kind of situations, MUI allows us to specify props into the styles object to produce different classes. [Check this out](https://material-ui.com/styles/basics/#adapting-based-on-props)

I copied some examples in the web for reference

```javascript
const useStyles = makeStyles<Theme, StyleProps>(theme => ({
  root: {
    background: 'green',
    height: isSticky => isSticky ? 'absolute' : 'relative',
  },
}));

export default function Hook() {

  const propsFromSomewhere = {
    isSticky: true
  }

  const classes = useStyles(props);
  return <Button className={classes.root}>Styled with Hook API</Button>;
}
```

```javascript
// Official documentation
const useStyles = makeStyles({
  // style rule
  foo: props => ({
    backgroundColor: props.backgroundColor,
  }),
  bar: {
    // CSS property
    color: props => props.color,
  },
});

```